app/frontend/src/features/search/SearchResult.tsx

```diff
@@ -83,26 +62,14 @@ export default function SearchResult({ user }: { user: User.AsObject }) {
               </Container>
             </Box>
 
-            <Container disableGutters className={classes.scoreBarContainer}>
-              <LinearProgress
-                variant="determinate"
-                value={user.communityStanding * 100}
-                className={classes.scoreBar}
-              />
-              <TextBody className={classes.scoreBarLabel}>
-                Community Standing
-              </TextBody>
-            </Container>
-            <Container disableGutters className={classes.scoreBarContainer}>
-              <LinearProgress
-                variant="determinate"
-                value={user.verification * 100}
-                className={classes.scoreBar}
-              />
-              <TextBody className={classes.scoreBarLabel}>
-                Verification Score
-              </TextBody>
-            </Container>
+            <ScoreBar
+              label="Community Standing"
+              value={user.communityStanding * 100}
```

Comment:
Would a `toPercentage` utility function be justified?

Reply:
Not really imo, if it's just *100